ledger/ledger-api-client/src/it/scala/com/digitalasset/ledger/client/LedgerClientIT.scala

```diff
@@ -64,6 +64,15 @@ final class LedgerClientIT
       }
     }
 
+    "get api version" in {
+      for {
+        client <- LedgerClient(channel, ClientConfiguration)
+        version <- client.versionClient.getApiVersion()
+      } yield {
+        version shouldBe a[String]
```

Comment:
Perhaps check that it matches some kind of meaningful pattern.

```suggestion
        version should fullyMatch regex """1\.\d+\.\d+"""
```